docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,24 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
```

Comment:
A Replica set can go dormant. If it's in charge of completing a migration, that would be awkward. The second suggestion is the better one though.